spec.html

```diff
@@ -7018,6 +7052,222 @@ <h1>
+      <emu-table id="table-disposecapability-record-fields" caption="DisposeCapability Record Fields">
+        <table>
+          <tr>
+            <th>
+              Field Name
+            </th>
+            <th>
+              Value
+            </th>
+            <th>
+              Meaning
+            </th>
+          </tr>
+          <tr>
+            <td>
+              [[DisposableResourceStack]]
+            </td>
+            <td>
+              A List of DisposableResource Records.
+            </td>
+            <td>
+              The resources to be disposed. Resources are added in the order they are initialized, and are disposed in reverse order.
+            </td>
+          </tr>
+        </table>
+      </emu-table>
+    </emu-clause>
+
+    <emu-clause id="sec-disposableresource-records">
+      <h1>DisposableResource Records</h1>
+      <p>A <dfn variants="DisposableResource Records">DisposableResource Record</dfn> is a Record value used to encapsulate a disposable object along with the method used to dispose the object. DisposableResource Records are produced by the CreateDisposableResource abstract operation.</p>
+      <p>DisposableResource Records have the fields listed in <emu-xref href="#table-disposableresource-record-fields"></emu-xref>:</p>
+      <emu-table id="table-disposableresource-record-fields" caption="DisposableResource Record Fields">
+        <table>
+          <tr>
+            <th>
+              Field Name
+            </th>
+            <th>
+              Value
+            </th>
+            <th>
+              Meaning
+            </th>
+          </tr>
+          <tr>
+            <td>
+              [[ResourceValue]]
+            </td>
+            <td>
+              an Object or *undefined*
+            </td>
+            <td>
+              The value to be disposed.
+            </td>
+          </tr>
+          <tr>
+            <td>
+              [[Hint]]
+            </td>
+            <td>
+              ~sync-dispose~
+            </td>
+            <td>
+              Indicates that the resource was added by a `using` declaration or DisposableStack object.
+            </td>
+          </tr>
+          <tr>
+            <td>
+              [[DisposeMethod]]
+            </td>
+            <td>
+              a function object
+            </td>
+            <td>
+              A function object that will be called with [[ResourceValue]] as its *this* value when the resource disposed.
+            </td>
+          </tr>
+        </table>
+      </emu-table>
+      <emu-note>
+        Currently, the only allowed value for [[Hint]] is ~sync-dispose~.
+      </emu-note>
+    </emu-clause>
+
+    <emu-clause id="sec-newdisposecapability" type="abstract operation">
+      <h1>NewDisposeCapability ( ): a DisposeCapability Record</h1>
+      <dl class="header"></dl>
```

Comment:
```suggestion
      <dl class="header">
      </dl>
```
is how we normally do an empty `<dl>`